AttributeError: 'str' object has no attribute 'joinpath'

Getting below error when calling d.dotslash("../logs") 

```python
  File "/home/hitul/coverfox/env/rblog/local/lib/python2.7/site-packages/speaklater.py", line 186, in __getattr__
    return getattr(self.value, name)
AttributeError: 'str' object has no attribute 'joinpath'
make: *** [run] Error 1
```

--- a/ChangeLog.rst
+++ b/ChangeLog.rst
@@ -6,6 +6,10 @@ master
 
 * don't auto configure ROOT_URLCONF if already configured
 
+0.5.0 - 30-Sep-2015
+-------------------
+* Fixed AttributeError when calling d.dotslash. 
+
 0.4.0 - 27-Aug-2015
 -------------------
 

--- a/ChangeLog.rst
+++ b/ChangeLog.rst
@@ -6,7 +6,7 @@ master
 
 * don't auto configure ROOT_URLCONF if already configured
 
-0.5.0 - 30-Sep-2015
+0.4.1 - 30-Sep-2015
 -------------------
 * Fixed AttributeError when calling d.dotslash. 
 

--- a/importd/__init__.py
+++ b/importd/__init__.py
@@ -60,6 +60,8 @@ try:
 except ImportError:
     resource = None
 
+from path import path
+
 start_time = datetime.now()
 if python_version().startswith('3'):
     basestring = unicode = str  # noqa lint:ok
@@ -344,7 +346,7 @@ class D(object):
 
     def _get_app_dir(self, pth):
         """Return the path of the app."""
-        return os.path.join(self.APP_DIR, pth)
+        return path(os.path.join(self.APP_DIR, pth))
 
     def dotslash(self, pth):
         """Mimic the unix './' behaviour."""

--- a/importd/__init__.py
+++ b/importd/__init__.py
@@ -4,7 +4,7 @@
 """ImportD django mini framework."""
 
 
-__version__ = "0.5.0"
+__version__ = "0.4.1"
 __license__ = "BSD"
 __author__ = "Amit Upadhyay"
 __email__ = "upadhyay@gmail.com"

--- a/importd/__init__.py
+++ b/importd/__init__.py
@@ -4,7 +4,7 @@
 """ImportD django mini framework."""
 
 
-__version__ = "0.4.0"
+__version__ = "0.5.0"
 __license__ = "BSD"
 __author__ = "Amit Upadhyay"
 __email__ = "upadhyay@gmail.com"

--- a/setup.py
+++ b/setup.py
@@ -50,7 +50,7 @@ setup(
 
     install_requires=[
         "fhurl>=0.1.7", "smarturls", "six", "Django>=1.3",
-        "dj-database-url",
+        "dj-database-url", "path.py"
     ],
 
 

--- a/tests/pathpy_test/main.py
+++ b/tests/pathpy_test/main.py
@@ -1,0 +1,19 @@
+# make sure to use importd from the repository
+import os
+import sys
+from importd import d
+
+d(
+    DEBUG=True,
+    LOG_LOCATION=d.dotslash("temp-log.py"),
+)
+
+if __name__ == "__main__":
+    path_with_py_api = os.path.join(os.path.dirname(os.path.abspath(__file__)), "temp-log.py")
+    path_wit_d = d.dotslash("temp-log.py")
+    if path_with_py_api == path_wit_d:
+        print("Test passed")
+    else:
+        raise Exception("Test failed")
+
+    d.main()

